device-registry/src/main/scala/org/genivi/sota/device_registry/DevicesResource.scala

```diff
@@ -120,9 +121,6 @@ class DevicesResource(namespaceExtractor: Directive1[Namespace],
         (get & path("groups") & pathEnd) {
           getGroupsForDevice(uuid)
         }
-      } ~
-      (get & pathEnd & parameter('namespace.as[Namespace])) { ns =>
-        searchDevice(ns)
```

Comment:
It is still here, but I moved it to the top so that it is in namespaceExtractor, so all routes that require the namespace are grouped together.